2009 smallcents

aDesertRat opened the pull request.

sw23 commented:
@aDesertRat Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like

aDesertRat commented:
Because of refactoring I had to reinstall and load my collections again. Found small cents was not displaying 2009 cents properly so I fixed it. Could not load many dimes and halfs so I wrote new classes for Roosevelt Dimes and Kennedy halfs with proofs etc. Also wrote a unified and complete Washington Quarters. I believe all are ready to merge although they need SharedTest entries. I'm not planning on writing any more classes until after you publish the yearly update which I hope will be soon. I may make some minor cosmetic changes until you freeze.

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

On Thursday, March 27th, 2025 at 9:28 PM, Spencer Williams ***@***.***> wrote:

> ***@***.***(https://github.com/aDesertRat) Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS2RR2FACTOB4RH7MND2WTFWZAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGE2DONBUGI).
> You are receiving this because you were mentioned.Message ID: ***@***.***>
>
> [sw23]sw23 left a comment [(anwilli5/coin-collection-android-US#239)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)
>
> ***@***.***(https://github.com/aDesertRat) Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS2RR2FACTOB4RH7MND2WTFWZAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGE2DONBUGI).
> You are receiving this because you were mentioned.Message ID: ***@***.***>

sw23 commented:
> Because of refactoring I had to reinstall and load my collections again. Found small cents was not displaying 2009 cents properly so I fixed it. Could not load many dimes and halfs so I wrote new classes for Roosevelt Dimes and Kennedy halfs with proofs etc. Also wrote a unified and complete Washington Quarters. I believe all are ready to merge although they need SharedTest entries. I'm not planning on writing any more classes until after you publish the yearly update which I hope will be soon. I may make some minor cosmetic changes until you freeze. Sent with [Proton Mail](https://proton.me/mail/home) secure email.
> […](#)
> On Thursday, March 27th, 2025 at 9:28 PM, Spencer Williams ***@***.***> wrote: ***@***.***(https://github.com/aDesertRat) Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like — Reply to this email directly, [view it on GitHub]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS2RR2FACTOB4RH7MND2WTFWZAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGE2DONBUGI). You are receiving this because you were mentioned.Message ID: ***@***.***> [sw23]sw23 left a comment [([anwilli5/coin-collection-android-US#239](https://github.com/anwilli5/coin-collection-android-US/pull/239))]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)) ***@***.***(https://github.com/aDesertRat) Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like — Reply to this email directly, [view it on GitHub]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS2RR2FACTOB4RH7MND2WTFWZAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGE2DONBUGI). You are receiving this because you were mentioned.Message ID: ***@***.***>

@aDesertRat ahhh, sorry to hear about your collections - if you happen to have a recent export backup file I'd be happy to manually patch it up so that you can restore from that. I will admit I have wiped out my collection a few times in the course of testing app updates that aren't fully ready.

We can go ahead and merge this and I can push some additional updates in a new PR. I see you commented out ProofSets and MintSets - I'm assuming this means you don't think we need these anymore, and we can keep just CoinSets?

aDesertRat commented:
Yes, dump mint sets and proof sets. Do you want me to rewrite coin sets to use checkboxs?

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

On Thursday, March 27th, 2025 at 11:09 PM, Spencer Williams ***@***.***> wrote:

>> Because of refactoring I had to reinstall and load my collections again. Found small cents was not displaying 2009 cents properly so I fixed it. Could not load many dimes and halfs so I wrote new classes for Roosevelt Dimes and Kennedy halfs with proofs etc. Also wrote a unified and complete Washington Quarters. I believe all are ready to merge although they need SharedTest entries. I'm not planning on writing any more classes until after you publish the yearly update which I hope will be soon. I may make some minor cosmetic changes until you freeze. Sent with [Proton Mail](https://proton.me/mail/home) secure email.
>> […](#)
>> On Thursday, March 27th, 2025 at 9:28 PM, Spencer Williams @.> wrote: @.(https://github.com/aDesertRat) Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like — Reply to this email directly, [view it on GitHub]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS2RR2FACTOB4RH7MND2WTFWZAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGE2DONBUGI). You are receiving this because you were mentioned.Message ID: @.> [sw23]sw23 left a comment [([anwilli5/coin-collection-android-US#239](https://github.com/anwilli5/coin-collection-android-US/pull/239))]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)) @.(https://github.com/aDesertRat) Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like — Reply to this email directly, [view it on GitHub]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS2RR2FACTOB4RH7MND2WTFWZAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGE2DONBUGI). You are receiving this because you were mentioned.Message ID: @.***>
>
> ***@***.***(https://github.com/aDesertRat) ahhh, sorry to hear about your collections - if you happen to have a recent export backup file I'd be happy to manually patch it up so that you can restore from that. I will admit I have wiped out my collection a few times in the course of testing app updates that aren't fully ready.
>
> We can go ahead and merge this and I can push some additional updates in a new PR. I see you commented out ProofSets and MintSets - I'm assuming this means you don't think we need these anymore, and we can keep just CoinSets?
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760321815), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS36V4N7EH7II2JTZJD2WTRRLAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGMZDCOBRGU).
> You are receiving this because you were mentioned.Message ID: ***@***.***>
>
> [sw23]sw23 left a comment [(anwilli5/coin-collection-android-US#239)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760321815)
>
>> Because of refactoring I had to reinstall and load my collections again. Found small cents was not displaying 2009 cents properly so I fixed it. Could not load many dimes and halfs so I wrote new classes for Roosevelt Dimes and Kennedy halfs with proofs etc. Also wrote a unified and complete Washington Quarters. I believe all are ready to merge although they need SharedTest entries. I'm not planning on writing any more classes until after you publish the yearly update which I hope will be soon. I may make some minor cosmetic changes until you freeze. Sent with [Proton Mail](https://proton.me/mail/home) secure email.
>> […](#)
>> On Thursday, March 27th, 2025 at 9:28 PM, Spencer Williams @.> wrote: @.(https://github.com/aDesertRat) Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like — Reply to this email directly, [view it on GitHub]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS2RR2FACTOB4RH7MND2WTFWZAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGE2DONBUGI). You are receiving this because you were mentioned.Message ID: @.> [sw23]sw23 left a comment [([anwilli5/coin-collection-android-US#239](https://github.com/anwilli5/coin-collection-android-US/pull/239))]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)) @.(https://github.com/aDesertRat) Are you still working on this one or is it ready to be merged in? It looks like some of these updates are ready - I can pull just those if you'd like — Reply to this email directly, [view it on GitHub]([#239 (comment)](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760147442)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS2RR2FACTOB4RH7MND2WTFWZAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGE2DONBUGI). You are receiving this because you were mentioned.Message ID: @.***>
>
> ***@***.***(https://github.com/aDesertRat) ahhh, sorry to hear about your collections - if you happen to have a recent export backup file I'd be happy to manually patch it up so that you can restore from that. I will admit I have wiped out my collection a few times in the course of testing app updates that aren't fully ready.
>
> We can go ahead and merge this and I can push some additional updates in a new PR. I see you commented out ProofSets and MintSets - I'm assuming this means you don't think we need these anymore, and we can keep just CoinSets?
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/anwilli5/coin-collection-android-US/pull/239#issuecomment-2760321815), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BJWPVS36V4N7EH7II2JTZJD2WTRRLAVCNFSM6AAAAABZW2FL6CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRQGMZDCOBRGU).
> You are receiving this because you were mentioned.Message ID: ***@***.***>

sw23 commented:
@aDesertRat I will merge this one since the unit tests are passing and it has fixes for other collections. After that, feel free to make the update if you have time. The updates in CollectionListInfo and one of the unit tests aren't well documented, but I can push a fix to your branch if needed for that
